src/include/homestore/homestore_decl.hpp

```diff
@@ -87,7 +87,7 @@ constexpr uint16_t MAX_UUID_LEN{128};
 static constexpr hs_uuid_t INVALID_SYSTEM_UUID{0};
 
 ///////////// All Enums //////////////////////////
-ENUM(HSDevType, uint8_t, Data, Fast);
+ENUM(HSDevType, uint8_t, Auto, Data, Fast);
```

Comment:
I am not clear on how HomeObj will specify the percentage for each service with this Auto devType in place.
How can HomeObj know whether it is mixed dev type, so that META+LOG+INDEX can accumulate upto 100% of the SSD drive, and the replication could consume 100% of the other devType drives, and if it is not mixed, say all HDD, or all NVMe, then META+LOG+INDEX+REPLICATION should accumulate upto 100%. 

Can you share how is HomeObject going to use this?